packages/propeller/src/Queue.ts

```diff
@@ -0,0 +1,37 @@
+class Node<T> {
```

Comment:
Lol at JS not having a built in queue implementation 🥴

Reply:
This is why we get people to code important complex algorithms stuff in interviews